### DISCUSSION TITLE
Chinese garbled characters are displayed

### DISCUSSION THREAD

**Discussion by Lifulingg** [QtExcel/QXlsx#242] [Ideas]
![image](https://user-images.githubusercontent.com/93233618/190993471-c723c627-6cda-4e01-b294-2e4760b25fea.png)

**Comment by j2doll** [comment]
Dear @Lifulingg 

Sorry for late reply.

Can you show us your code and excel file?

Also, because I don't know Chinese, please let me know what is normal.

**Reply by MoNianYin** [reply]
check your codec for QString

**Reply by david980508** [reply]
But this porblem can't be reproduced,gives env or files to track further
